README.md

```diff
@@ -84,6 +89,27 @@ while true; do kubectl --namespace tika-test port-forward $POD_NAME 9998:$CONTAI
 * Install it:
   - with Helm 3: `helm install tika . --set image.tag=latest-full`
 
+### Custom configuration for tika
+
+To use custom [configuration](https://tika.apache.org/1.9/configuring.html) values for apache tika, use the `tikaConfig` key in the `values.yaml`.
+Example:
+```
+tikaConfig: |
+  <?xml version="1.0" encoding="UTF-8"?>
+  <properties>
+    <parsers>
+      <!-- Default Parser for most things, except for 2 mime types -->
+      <parser class="org.apache.tika.parser.DefaultParser">
+        <mime-exclude>image/jpeg</mime-exclude>
+        <mime-exclude>application/pdf</mime-exclude>
+      </parser>
+      <!-- Use a different parser for PDF -->
```

Comment:
why do we want this empty parser here?

Reply:
Good catch, we should just keep the default.

Reply:
Thanks for the review. I've updated it.